numpy/_configtool.py

```diff
@@ -0,0 +1,38 @@
+import argparse
+from pathlib import Path
+import sys
+
+from .version import __version__
+from .lib._utils_impl import get_include
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--version",
+        action="version",
+        version=__version__,
+        help="Print the version and exit.",
+    )
+    parser.add_argument(
+        "--includes",
```

Comment:
It's more typical to see `--cflags` here. I know pybind11-config doesn't use that, and I just swallowed my comments and implemented the existing interface, but...

Reply:
Sure, I'll update to `--cflags`, thanks.